internal/pkg/template/templates/app/app.yml

```diff
@@ -30,7 +30,11 @@ Conditions:
 Resources:
   AdministrationRole:
     Metadata:
+      {{- if .PermissionsBoundary}}
+      'aws:copilot:description': 'A StackSet admin role with permissions boundary {{.PermissionsBoundary}} assumed by CloudFormation to manage regional stacks'
```

Comment:
Just a one-liner alternative as there are many places where similar code is there. 
```suggestion
      'aws:copilot:description':  {{ if .PermissionsBoundary}} 'A StackSet admin role with permissions boundary {{.PermissionsBoundary}} assumed by CloudFormation to manage regional stacks' {{ else}} 'A StackSet admin role assumed by CloudFormation to manage regional stacks' {{ end }}
```

Reply:
Can we possibly make it a function that takes a string and output a processed string when `{{ if .PermissionsBoundary}}`? The benefit is that we don't need to repeat ourselves whenever we add new roles or want to modify the description.

Reply:
Yeah, I'll think about it and possibly do it in a different PR!